lib/filterURLPassword.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { URL } = require('url');
+
+/**
+ * filter the auth of url string
+ * @param {URL} input filtered url
+ */
+module.exports = function filterURLPassword(input) {
+  const url = new URL(input);
```

Comment:
how about use legacy API `url.parse`.

egg-mongoose require 6.x, and URL is intro after 6.13. I'm not sure if it's break.

Reply:
**The status of `ur.parse` is `Stability: 0 - Deprecated`.**
https://nodejs.org/docs/latest/api/url.html#url_legacy_url_api

## Suggest
1. Check node version with `process.version` and switch function between url.parse or URL class.
2. Specify node version like `{ "engines" : { "node" : ">=6.13.0" } }`.
3. Refactor the filterURLPassword, from node API to JS API, such like Regex.

Reply:
@atian25 What do you think?

Reply:
node 11 is not LTS,  we will change it later.